{{cookiecutter.github_repository}}/ansible.cfg

```diff
@@ -0,0 +1 @@
+provisioner/ansible.cfg
```

Comment:
we might not need this as provisioner is run in the context of `provisioner/` folder always. Thoughts, I prefer to have as less file possible in root directory.